include/rocksdb/utilities/options_type.h

```diff
@@ -57,6 +57,7 @@ enum class OptionType {
   kEncodedString,
   kTemperature,
   kArray,
+  kPrepopulateBlobCache,
```

Comment:
Isn't this `kEnum`?

Reply:
Thanks! I will double check.

Reply:
This is pretty interesting. Looks like many people didn't realize we have `kEnum` now. For example, `kTemperature` shouldn't be added here. In this PR, I will only fix  `kPrepopulateBlobCache`, but I can make a PR to fix others.

Reply:
Done